packages/rolldown/src/plugin/bindingify-build-hooks.ts

```diff
@@ -46,10 +48,16 @@ export function bindingifyBuildStart(
 
   return {
     plugin: async (ctx) => {
-      await handler.call(
-        new PluginContext(options, ctx, plugin, pluginContextData),
-        options,
-      )
+      try {
+        await handler.call(
+          new PluginContext(options, ctx, plugin, pluginContextData),
+          options,
+        )
+      } catch (e: any) {
+        return error(
+          logPluginError(e, plugin.name ?? '<unknown>', { hook: 'buildStart' }),
```

Comment:
Not sure the overhead of wrapping function multiple times. If the overhead is acceptable, we should inject these helpers automatically.

Reply:
All these `try/catch` boilerplate doesn't look nice, but I couldn't find a concise abstraction. 
Now I'm thinking that maybe I can wrap it just before returning `bindingifyPlugin` :thinking: Let me try.

Reply:
I added a wrapper in a single place `bindingifyPlugin` https://github.com/rolldown/rolldown/pull/2767/commits/c001691a481adcd3e8400b8464960da47f3b14fe
Not completely sure but I don't think one more async/await call chain affects overall perf significantly.